### PR TITLE
chore(localization): update strings in registration page (resolves #792)

### LINKED ISSUE #792: Update text for Registration
https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=4%3A137

### PR DESCRIPTION
**Pull request by chosww**
See #792 for details.

### CODE REVIEW

File: resources/views/dashboard.blade.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
                 </div>
             </x-invitation>
         @endif
-        <p>{{ $user->name }}@if($memberable), {{ $memberable->name }}@endif</p>
```
Comment (greatislander) [inline resources/views/dashboard.blade.php:17]: The `@if` tag here no longer does anything— it needs to be wrapped around the `{{ $memberable->name }}` (this accounts for the test failures as some users are never members of an organization).

**Review by greatislander — CHANGES_REQUESTED** [review]

File: resources/views/auth/register/steps/4.blade.php [hunk 2]
```diff
@@ -7,7 +7,13 @@
     <div class="field @error('password') field--error @enderror stack">
         <x-hearth-label for="password" :value="__('hearth::auth.label_password')" />
         <x-hearth-hint for="password">
-            {{ __('Passwords must be at least eight characters in length and include at least one uppercase letter, at least one number, and at least one special character.') }}
+            {{ __('For your security, please make sure your password has:') }}
```
Comment (greatislander) [inline resources/views/auth/register/steps/4.blade.php]: the `<x-hearth-hint>` component doesn't support multiple block-level elements (in this case, a paragraph and an unordered list). It's probably best to not use the hint component and wrap this all in a `<div>` like this:

```html
<div class="field__hint" id="password-hint">
...
</div>
``

File: resources/views/organizations/show-role-selection.blade.php [hunk 3]
```diff
@@ -4,13 +4,12 @@
...
         </h1>
-
-        <h2>{{ __('Create new community organization') }}</h2>
     </x-slot>
 
     <form class="stack" action="{{ localized_route('organizations.save-roles', $organization) }}" method="post" novalidate>
         <fieldset class="field @error('type') field--error @enderror">
-            <legend>{{ __('What type of organization are you?') }}</legend>
+            <legend>{{ __('Your organization’s role') }}</legend>
+            <x-hearth-hint for="roles">{{ __('Please tell us what your organization would like to do here. You must pick at least one of these roles. You can always change this later.') }} <a target="_blank" href="{{ localized_route('about.for-community-organizations') }}">Learn more about these roles.</a> </x-hearth-hint>
```
Comment (greatislander) [inline resources/views/organizations/show-role-selection.blade.php]: "Learn more about these roles" is not properly wrapped in `__('')` and thus won't be picked up for translation. Please make sure it is processed like line 11 above.

**Review by greatislander — CHANGES_REQUESTED** [review]

File: resources/views/organizations/show-role-selection.blade.php [hunk 4]
```diff
@@ -9,7 +9,7 @@
     <form class="stack" action="{{ localized_route('organizations.save-roles', $organization) }}" method="post" novalidate>
         <fieldset class="field @error('type') field--error @enderror">
             <legend>{{ __('Your organization’s role') }}</legend>
-            <x-hearth-hint for="roles">{{ __('Please tell us what your organization would like to do here. You must pick at least one of these roles. You can always change this later.') }} <a target="_blank" href="{{ localized_route('about.for-community-organizations') }}">Learn more about these roles.</a> </x-hearth-hint>
+            <x-hearth-hint for="roles">{{ __('Please tell us what your organization would like to do here. You must pick at least one of these roles. You can always change this later.') }} <a target="_blank" href="{{ localized_route('about.for-community-organizations') }}">{{ __('Learn more about these roles') }}</a> </x-hearth-hint>
```
Comment (greatislander) [inline resources/views/organizations/show-role-selection.blade.php:12]: I think you are missing a period at the end of this sentence.

File: resources/views/auth/register/steps/4.blade.php [hunk 5]
```diff
@@ -6,15 +6,15 @@
     <!-- Password -->
     <div class="field @error('password') field--error @enderror stack">
         <x-hearth-label for="password" :value="__('hearth::auth.label_password')" />
-        <x-hearth-hint for="password">
+        <div class="field__hint" id="password-hint">
             {{ __('For your security, please make sure your password has:') }}
```
Comment (greatislander) [inline resources/views/auth/register/steps/4.blade.php]: Wrap this in a `<p>` tag as we aren't processing it with Markdown as is normally the case in the hint component.

**Review by greatislander — CHANGES_REQUESTED** [review]

**Review by greatislander — APPROVED** [review]

**Comment by cherylhjli** [comment]
@chosww what page did you update? Sorry the original figma link doesn't seem to work anymore!

**Comment by cherylhjli** [comment]
@chosww the copy under the "learn more" component on this page isn't there: https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1125%3A4237

**Pull request merged**